### PR TITLE
RFC 177 - fetch_json and other ShadowRealm facilities

### PR DESCRIPTION
**Pull request by ptomato**
Request for feedback on the APIs proposed in
https://github.com/web-platform-tests/wpt/pull/43639 adding some facilities for executing existing tests inside ShadowRealm global contexts.

### CODE REVIEW

File: rfcs/fetch_json_et_al.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+The most notable change is a `fetch_json` function to allow importing
+test data from JSON files across the sandboxing boundary.
+
+## Details
+
+The [pull request][pr] comprises four commits, which I'll explain in
+detail in this section.
+
+### Document self.GLOBAL.isShadowRealm()
```
Comment (foolip) [inline rfcs/fetch_json_et_al.md:25]: If you want to land this fix right away and drop it from this RFC that would be fine.

**Review by foolip — APPROVED** [review]

File: rfcs/fetch_json_et_al.md (hunk 1 above)
Comment (ptomato) [reply rfcs/fetch_json_et_al.md:25]: Thanks, I may split it out if I get a chance before the RFC finishes.

**Comment by ptomato** [comment]
Just checking in about the process — can we consider this accepted or is it wise to wait another week to allow folks to come back from winter holidays?

**Comment by ptomato** [comment]
@foolip At this point, can we land this?

**Comment by foolip** [comment]
@ptomato sure thing, per our process enough time has passed, this RFC has passed.

**Pull request merged**